docs/adr/0002-plantuml-diagrams-as-code.md

```diff
@@ -0,0 +1,29 @@
+# 0002- Use PlantUML as our diagrams as code tool
+
+2023-01-24
+
+## Status
+
+Accepted
+
+## Context
```

Comment:
I've made various changes to this based on what you've said. Here's my thoughts on each of the sections.

**Context**
I've updated this so its more about 'setting the scene'. Why we needed to make the decision like you stated there, this included benefits of diagrams as code (since that decision had already been made it was part of the context).

**Decision**
This is now just about why we chose to use PlantUML over the alternatives we had investigated.

**Consequence**
The consequences, good or bad as to things that have happened as a result of this change.